sdk/communication/Azure.Communication.CallingServer/tests/CallConnection/CallConnectionLiveTests.cs

```diff
@@ -101,6 +107,359 @@ public async Task RunCreateAddRemoveHangupScenarioTests()
+                await WaitForOperationCompletion().ConfigureAwait(false);
+                await UnmuteParticipantOperation(callConnection, userId).ConfigureAwait(false);
+
+                // Get Participant
+                await WaitForOperationCompletion().ConfigureAwait(false);
+                var getUnmutedParticipant = await GetParticipantOperation(callConnection, userId).ConfigureAwait(false);
+                Assert.IsTrue(getUnmutedParticipant.IsMuted == false);
+
+                // Get Participants
+                await WaitForOperationCompletion().ConfigureAwait(false);
+                var getParticipants = await GetParticipantsOperation(callConnection).ConfigureAwait(false);
+                Assert.IsTrue(getParticipants.Count() > 2);
+
+                // Remove Participant
+                await WaitForOperationCompletion().ConfigureAwait(false);
+                await RemoveParticipantOperation(callConnection, userId).ConfigureAwait(false);
+            }
+            catch (RequestFailedException ex)
+            {
+                Console.WriteLine(ex.Message);
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+            finally
+            {
+                // Hang up the Call, there is one call leg in this test case, hangup the call will also delete the call as the result.
+                await WaitForOperationCompletion().ConfigureAwait(false);
+                await HangupOperation(callConnection).ConfigureAwait(false);
+            }
+        }
+
+        [Test]
+        public async Task RunCreateAddHoldResumeAudioRemoveHangupScenarioTests()
+        {
+            if (SkipCallingServerInteractionLiveTests)
+                Assert.Ignore("Skip callingserver interaction live tests flag is on.");
+
+            CallingServerClient client = CreateInstrumentedCallingServerClientWithConnectionString();
+
+            // Establish a call
+            var callConnection = await CreateCallConnectionOperation(client).ConfigureAwait(false);
+
+            try
+            {
+                string userId = GetFixedUserId(TestEnvironment.UserIdentifier);
+
+                // Add Participant
+                await WaitForOperationCompletion().ConfigureAwait(false);
+                AddParticipantResult addParticipantResult = await AddParticipantOperation(callConnection, userId).ConfigureAwait(false);
+                Assert.NotNull(addParticipantResult);
+
+                // Get Call
+                await WaitForOperationCompletion().ConfigureAwait(false);
+                var getCallConnection = await GetCallOperation(callConnection).ConfigureAwait(false);
+                Assert.AreEqual(getCallConnection.CallConnectionId, callConnection.Value.CallConnectionId);
+
+                // Hold Participant
```

Comment:
here and at other places, pl make sure the comment reflects the line beside it or right under it.

Reply:
Made changes